src/main/java/com/ldtteam/structurize/optifine/OptifineCompat.java

```diff
@@ -104,6 +117,20 @@ private void setupReflectedMethodReferences() throws ClassNotFoundException, NoS
         final Class<?> shaderRenderClass = Class.forName("net.optifine.shaders.ShadersRender");
         final Class<?> sVertexBuilderClass = Class.forName("net.optifine.shaders.SVertexBuilder");
         final Class<?> shadersClass = Class.forName("net.optifine.shaders.Shaders");
+        final Class<?> shaderUniform3fClass = Class.forName("net.optifine.shaders.uniform.ShaderUniform3f");
+
+        final Class<RenderSystem> renderSystemClass = RenderSystem.class;
+        final Class<FogRenderer> fogRendererClass = FogRenderer.class;
+        final Class<MultiBufferSource> multiBufferSourceClass = MultiBufferSource.class;
+
+        isFogOffMethod = configClass.getMethod("isFogOff");
+        isFogOffMethod.setAccessible(true); 
+
+        fogStandardField = fogRendererClass.getField("fogStandard");
+        fogStandardField.setAccessible(true);
+        
+        setFogAllowedMethod = renderSystemClass.getMethod("setFogAllowed", boolean.class);
+        setFogAllowedMethod.setAccessible(true);
```

Comment:
This won't work in production, and you really should not need it either.....

Reply:
These are optifine methods/fields in vanilla class, why should it not work?

Reply:
Ah
Optifine adds them...........Oef....